modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -315,6 +315,5 @@ routes:
 [IMPORTANT]
 ====
 You cannot use the OVN-Kubernetes `br-ex` bridge as the next hop interface when configuring a static route unless you manually configured a customized `br-ex` bridge.
```

Comment:
We'll need to remove the "unless you manually configured a customized `br-ex` bridge" part here too.